src/GameLoop.tsx

```diff
@@ -55,223 +59,214 @@ const stateMachine = createMachine({
 
 // Handle states.
 const stateHandler = interpret(stateMachine).onTransition((state) => {
-    // TODO
+    console.log("   STATE:", state.value);
 });
 stateHandler.start();
 
-let placedCells = new Set();
+// Various game logic vars.
+const FPS =
```

Comment:
Why do you need this?

Reply:
FPS is used to calculate any game logic catchup required.

Reply:
We don't know the actual FPS though.

Reply:
You're right, we cannot know the player's FPS. However, the `FPS` variable is more about limiting the FPS of players with faster machines to the given count. I've changed the var name to reflect this

Reply:
What non discrete animation are we doing?

Reply:
I'm not sure why that matters here, could you clarify?

Reply:
Why do we need `setAnimationFrame` in the first place?

Reply:
Did you mean `requestAnimationFrame`? The reason why is to run a central loop that updates both game logic and rendering.

Reply:
Yeah sorry, typo.  What about `setInterval`?

Reply:
`setInterval` is not as precise. [Here's](https://inventinggames.github.io/posts/gameloopjs/) a demonstration

Reply:
I think it should be precise enough for our ticks.  Aren't they more on the order of a second?  But we can see.

Reply:
We'll see if it needs to be changed